changelogs/fragments/9186-fix-broken-check-mode-in-github-key.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - github_key - when check_mode was set, a faulty call to ```datetime.strftime(...)``` was being made which generated an exception. (https://github.com/ansible-collections/community.general/issues/9185)
```

```suggestion
  - github_key - in check mode, a faulty call to ```datetime.strftime(...)``` was being made which generated an exception (https://github.com/ansible-collections/community.general/issues/9185).
```